template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hive/HiveMetastoreConfigProviderTest.java

```diff
@@ -284,6 +285,27 @@ void getServiceConfigsTestDatalakeWithLdap() {
         assertThat(configNameToVariableNameMap).isEmpty();
     }
 
+    @Test void getServiceConfigsTestDatahubAndNoLdap() {
```

Comment:
Could you please add one more test case for the "Data Hub & initiator == on" case (i.e. when using CM < 7.1.1)?

Reply:
Done!